pkgs/development/python-modules/audio-metadata/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi
+, attrs
+, bidict
+, bitstruct
+, more-itertools
+, pprintpp
+}:
+
+buildPythonPackage rec {
+  pname = "audio-metadata";
+  version = "0.2.0";
+
```

Comment:
I don't see python2 support declared https://github.com/thebigmunch/audio-metadata/blob/d53cf68258fba959663945625eefff4fcb413280/setup.py#L25

so it needs to be disabled for that interpreter with the `disabled` parameter.

Reply:
Actual constraint is here https://github.com/thebigmunch/audio-metadata/blob/d53cf68258fba959663945625eefff4fcb413280/pyproject.toml#L20

Reply:
Thank you! I couldn't figure out how to disable python2.